google/cloud/bigquery/job/base.py

```diff
@@ -171,6 +171,33 @@ def __setattr__(self, name, value):
             )
         super(_JobConfig, self).__setattr__(name, value)
 
+    @property
+    def jobtimeout_ms(self):
+        """ Optional parameter. Job timeout in milliseconds. If this time limit is exceeded, BigQuery might attempt to stop the job.
+
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfiguration.FIELDS.job_timeout_ms
+
+        e.g.
+            job_config = bigquery.QueryJobConfig( jobtimeout = 5000 )
+            
+                or
+
+            job_config.jobtimeout = 5000
+        Raises:
+            ValueError: If ``value`` type is invalid.
+        """
+        
+        # None as this is an optional parameter.
```

Comment:
It could be `None`, but it could also be a real integer value. We should return `self._properties.get("jobTimeoutMs")` (converted to an integer if not None. I believe there are other integer properties that use a helper function for this conversion.)